doc/source/release-notes/Percona-Server-8.0.15-6.rst

```diff
@@ -0,0 +1,71 @@
+.. rn:: 8.0.15-6
+
+================================================================================
+|Percona Server| |release|
+================================================================================
+
+|Percona| announces the release of |Percona Server| |release| on |date|
+(downloads are available `here
+<https://www.percona.com/downloads/Percona-Server-8.0/>`__ and from the `Percona
+Software Repositories
+<https://www.percona.com/doc/percona-server/8.0/installation.html#installing-from-binaries>`__).
+
+This release includes fixes to bugs found in previous releases of |Percona
+Server| 8.0.
+
+.. admonition:: Incompatible changes
+
+   In previous releases, the audit log used to produce time stamps
+   inconsistent with the ISO 8601 standard. Release |release| of
+   |Percona Server| solves this problem. This change, however, may
+   break programs that rely on the old time stamp format.
+
+.. include:: ../.res/text/encrypt_binlog.removing.txt
+
+This release is based on |MySQL| 8.0.14 and 8.0.15. It includes all
+bug fixes in these releases. |Percona Server| :rn:`8.0.14` was skipped.
+
+|Percona Server| |release| is now the current GA release in the 8.0
+series. All of |Percona|’s software is open-source and free.
+
+Percona Server for MySQL 8.0 includes all the `features available in MySQL 8.0
+Community Edition
+<https://dev.mysql.com/doc/refman/8.0/en/mysql-nutshell.html>`__ in addition to
+enterprise-grade features developed by Percona.  For a list of highlighted
+features from both MySQL 8.0 and Percona Server for MySQL 8.0, please see the
+`GA release announcement
+<https://www.percona.com/blog/2018/12/21/announcing-general-availability-of-percona-server-for-mysql-8-0/>`__.
+
+.. note::
+
+   If you are upgrading from 5.7 to 8.0, please ensure that you read the
+   `upgrade guide
+   <https://www.percona.com/doc/percona-server/8.0/upgrading_guide.html>`__ and
+   the document `Changed in Percona Server for MySQL 8.0
+   <https://www.percona.com/doc/percona-server/8.0/changed_in_version.html>`__.
+
+New Features
+================================================================================
+
+- new session :variable:`rocksdb_disable_file_deletions` MyRocks variable has
```

Comment:
Instead of copy pasting var descriptions, I'd rather say "The server part of MyRocks cross-engine consistent physical backups has been implemented by introducing two new vars X and Y"